lib/inets/src/http_client/httpc_handler.erl

```diff
@@ -305,8 +305,9 @@ terminate(normal,
 %% Socket closed remotely
 terminate(normal, 
           #state{session = #session{socket      = {remote_close, Socket},
-                                    socket_type = SocketType, 
-                                    id          = Id}, 
+                                    socket_type = SocketType,
+                                    type        = Type,
+                                    id          = Id},
```

Comment:
Would be nice if the variable matchings here could be in the same order as on line 334-337, That would make it easier to realize that the two function clauses are almost identical except for the last few lines.